procedural/timeline/src/main/kotlin/gov/nasa/ammos/aerie/procedural/timeline/collections/profiles/Numbers.kt

```diff
@@ -223,5 +223,38 @@ data class Numbers<N: Number>(private val timeline: Timeline<Segment<N>, Numbers
       }
       seg.withNewValue(number)
     }) }
+
+    /**
+     * Converts a list of serialized value segments into an integer [Numbers] profile;
+     * for use with [gov.nasa.ammos.aerie.procedural.timeline.plan.Plan.resource].
+     *
+     * @throws ArithmeticException if any of the segment values are not integers
+     */
+    @JvmStatic fun intDeserializer() = { list: List<Segment<SerializedValue>> -> Numbers(list.map { seg ->
```

Comment:
The new deserializer methods in this class are unused, so I'm not totally sure how they're related to the rest of this PR